client/objects.lua

```diff
@@ -199,25 +106,29 @@ RegisterNetEvent('police:client:spawnObject', function(objectId, type, player)
 end)
 
 RegisterNetEvent('police:client:SpawnSpikeStrip', function()
-    if #SpawnedSpikes + 1 < Config.MaxSpikes then
-        if PlayerJob.name == "police" and PlayerJob.onduty then
-            local spawnCoords = GetOffsetFromEntityInWorldCoords(PlayerPedId(), 0.0, 2.0, 0.0)
-            local spike = CreateObject(spikemodel, spawnCoords.x, spawnCoords.y, spawnCoords.z, 1, 1, 1)
-            local netid = NetworkGetNetworkIdFromEntity(spike)
-            SetNetworkIdExistsOnAllMachines(netid, true)
-            SetNetworkIdCanMigrate(netid, false)
-            SetEntityHeading(spike, GetEntityHeading(PlayerPedId()))
-            PlaceObjectOnGroundProperly(spike)
-            SpawnedSpikes[#SpawnedSpikes+1] = {
-                coords = vector3(spawnCoords.x, spawnCoords.y, spawnCoords.z),
-                netid = netid,
-                object = spike,
-            }
-            TriggerServerEvent('police:server:SyncSpikes', SpawnedSpikes)
-        end
-    else
+    if #SpawnedSpikes >= Config.MaxSpikes then
+        lib.notify({ description = Lang:t("error.no_spikestripe"), type = 'error'})
+        return
+    end
+
+    if PlayerJob.name ~= "police" or not PlayerJob.onduty then
         QBCore.Functions.Notify(Lang:t("error.no_spikestripe"), 'error')
```

Comment:
```suggestion
        lib.notify({ description = Lang:t("error.no_spikestripe"), type = 'error' })
```
Change to ox_lib notification.

Reply:
Consider combining these two ifs with an or condition as the body of both are the same

Reply:
True that, didn't notice